### PR TITLE
Dec 2019 update

### PR DESCRIPTION
**Pull request by givenscj**
Update images from azure portal.  Address repo issues.

### CODE REVIEW

File: Hands-on lab/HOL step-by-step - Internet of Things.md [hunk 1]
```diff
@@ -142,21 +144,27 @@ If you want to save this connection string with your project (in case you stop d
...
+> NOTE:  If the form does not display, it is due to the `TODO` tasks in the code.  You can remove them in the following steps or simply add the following code to the `MainForm.Designer.cs` on line 260
+
+```csharp
+this.txtIotHubCnString.Text = "YOUR CONNECTION STRING";
+```
+
    ![In the Visual Studio Solution Explorer window, SmartMeterSimulator is expanded, and under it, MainForm.cs is highlighted.](media/visual-studio-solution-explorer-mainform-cs.png 'Visual Studio Solution Explorer')
 
-3. In the Windows Forms designer surface, click the **IoT Hub Connection String TextBox** to select it.
+1. In the Windows Forms designer surface, select the **IoT Hub Connection String TextBox** to select it.
```
Comment (emilysaeli) [inline Hands-on lab/HOL step-by-step - Internet of Things.md]: This numbering is wrong. Either a new task should start above or it should continue at four.

File: Hands-on lab/HOL step-by-step - Internet of Things.md [hunk 2]
```diff
@@ -916,59 +931,61 @@ In this task, you will create a new Databricks notebook to perform some processi
    dbutils.widgets.text("accountKey", "", "Account Key")
    ```
 
-5. Now, select the Run button on the right side of the cell, and select **Run cell**.
+> **Note**:  Make sure to be aware of any indents\tabs. Python  treats indents\tabs with specific syntactical meaning.
+
+1. Now, select the Run button on the right side of the cell, and select **Run cell**.
```
Comment (emilysaeli) [inline Hands-on lab/HOL step-by-step - Internet of Things.md]: This numbering is wrong. Either a new task should start above or it should continue at five.

**Comment by emilysaeli** [comment]
WDS trainer presentation ppt has font smaller than 18 pt, static images. Please fix and resubmit. 
Slide 18 has a spelling error in a picture I can't fix (managementWebsite should be "management website")

File: Hands-on lab/HOL step-by-step - Internet of Things.md (hunk 1 above)
Comment (timahenning) [reply Hands-on lab/HOL step-by-step - Internet of Things.md]: This has been fixed.

**Pull request merged**